packages/graphiql-react/src/explorer/components/search.tsx

```diff
@@ -50,10 +50,11 @@ export function Search() {
 
   useEffect(() => {
     function handleKeyDown(event: KeyboardEvent) {
-      if (event.metaKey && event.keyCode === 75 && inputRef.current) {
-        inputRef.current.focus();
+      if (event.metaKey && (event.key === 'K' || event.key === 'k')) {
```

Comment:
catch both lowercase and uppercase (as current behaviour)

Reply:
I'd be fine to changing this to just lowercase, I'd guess that's how most users use this shortcut